Missing serializer in about view

**Describe the bug**
A warning message is logged when request about view because the serializer is not defined. 
```
2024-04-03 14:50:35,360 [WARNING] [MainProcess] view's AboutView raised exception during schema generation; use `getattr(self, 'swagger_fake_view', False)` to detect and short-circuit this
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/drf_yasg/inspectors/base.py", line 42, in call_view_method
    return view_method()
           ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/generics.py", line 108, in get_serializer
    serializer_class = self.get_serializer_class()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/generics.py", line 122, in get_serializer_class
    assert self.serializer_class is not None, (
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 'AboutView' should either include a `serializer_class` attribute, or override the `get_serializer_class()` method.

```

--- a/safe_locking_service/locking_events/serializers.py
+++ b/safe_locking_service/locking_events/serializers.py
@@ -14,6 +14,15 @@ from safe_locking_service.locking_events.models import (
 from safe_locking_service.locking_events.services.locking_service import EventType
 
 
+class AboutSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    version = serializers.CharField()
+    api_version = serializers.CharField()
+    secure = serializers.BooleanField()
+    host = serializers.CharField()
+    headers = serializers.ListField()
+
+
 class EventTypeSerializer(serializers.Serializer):
     event_type = serializers.SerializerMethodField()
 

--- a/safe_locking_service/locking_events/views.py
+++ b/safe_locking_service/locking_events/views.py
@@ -21,6 +21,7 @@ from safe_locking_service.locking_events.pagination import (
     SmallPagination,
 )
 from safe_locking_service.locking_events.serializers import (
+    AboutSerializer,
     AllEventsDocSerializer,
     LeaderBoardSerializer,
     LockEventSerializer,
@@ -35,17 +36,23 @@ class AboutView(GenericAPIView):
     Returns information and configuration of the service
     """
 
-    @method_decorator(cache_page(5 * 60))  # 5 minutes
-    def get(self, request, format=None):
-        content = {
+    serializer_class = AboutSerializer
+
+    def get_queryset(self):
+        return {
             "name": "Safe Locking Service",
             "version": __version__,
-            "api_version": request.version,
-            "secure": request.is_secure(),
-            "host": request.get_host(),
-            "headers": [x for x in request.META.keys() if "FORWARD" in x],
+            "api_version": self.request.version,
+            "secure": self.request.is_secure(),
+            "host": self.request.get_host(),
+            "headers": [x for x in self.request.META.keys() if "FORWARD" in x],
         }
-        return Response(content)
+
+    @method_decorator(cache_page(5 * 60))  # 5 minutes
+    def get(self, request, format=None):
+        serializer = self.serializer_class(data=self.get_queryset())
+        serializer.is_valid()
+        return Response(serializer.data)
 
 
 class AllEventsView(ListAPIView):

